build_tooling/prep_cpp_build.sh

```diff
@@ -22,6 +22,8 @@ case `uname -a` in
             mkdir "${ARCTICDB_BUILD_DIR:?environment variable is not set}"
             MSYS=winsymlinks:nativestrict ln -s "$ARCTICDB_BUILD_DIR" out
         fi
+        export ARCTICDB_VCPKG_INSTALL_DIR="C:\test_arcticdb_install_dir"  # TODO: should configure in build.yml
```

Comment:
Minor: Use `/` instead of `\` because the latter is the escape character.

Reply:
Done